lib/matplotlib/quiver.py

```diff
@@ -451,11 +461,10 @@ class Quiver(mcollections.PolyCollection):
     """
     Specialized PolyCollection for arrows.
 
-    The only API method is set_UVC(), which can be used
-    to change the size, orientation, and color of the
-    arrows; their locations are fixed when the class is
-    instantiated.  Possibly this method will be useful
-    in animations.
+    The only API methods are ``set_UVC()``, ``set_XY``,
```

Comment:
Remove "only"? (Does three count as only?)

Single back-ticks will link, right? I guess one should be consistent with either using `()` for all or none of the methods.